backend/app/Http/Controllers/MarketplaceController.php

```diff
@@ -54,6 +56,32 @@ public function updateCard($card_id)
         return $this->getCardDetail($card_id);
     }
 
+    public function putTransaction($card_id)
+    {
+        //update card user's id with current id
+        $user = auth()->user();
+        $card = Card::find($card_id);
+        $listing = Listing::where('card_id', $card_id)->first();
+        if ($card->isUserOwner($user)) {
```

Comment:
What is this supposed to do?

Reply:
It is for the buy button, a check if the user's own the card they can't buy it again

Reply:
The response message below says: `'User does not own the card'`

`return response()->build(self::RESPONSE_MESSAGE_ERROR_UNAUTHORIZED, 'User does not own the card');`

Reply:
oh it should be "User already owns the card". I will change that